dill/_dill.py

```diff
@@ -980,24 +900,6 @@ def _create_code(*args):
             return CodeType(args[0], *argz)
         elif len(args) == 15: return CodeType(*args)
         return CodeType(args[0], 0, *args[1:]) # from 2.7
-    # obj.co_argcount, obj.co_nlocals, obj.co_stacksize, obj.co_flags,
```

Comment:
We should either outdent the last `elif` (for python 3.7), or convert it to an `else`.

Reply:
You are right, I missed that there was an implicit else statement. I'll will just convert the last `elif` to an `else` then.